redaxo/src/core/lib/console/command_loader.php

```diff
@@ -17,6 +17,7 @@ public function __construct()
         $commands = [
             'cache:clear' => rex_command_cache_clear::class,
             'db:connection-options' => rex_command_db_connection_options::class,
+            'db:set-connection' => rex_command_db_set_connection::class,
```

Comment:
vllt. analog dem anderen db commando?

```suggestion
            'db:set-connection-options' => rex_command_db_set_connection::class,
```